gui/lib/Help.elm

```diff
@@ -124,6 +124,33 @@ view helpers model =
                     ]
                 ]
             , h2 [] [ text (helpers.t "Help Official Support") ]
+            , if model.status == Success then
+                p [ class "message--success" ]
+                    [ text (helpers.t "Help Your mail has been sent. We will try to respond to it really soon!") ]
```

Comment:
You can close and reopen the window ;-)
I think we can improve this after the redesign of the GUI.